src/main/resources/default/view/parts/singleSelect.html

```diff
@@ -5,10 +5,10 @@
 *  Copyright by scireum GmbH
 *  http://www.scireum.de - info@scireum.de
 *@
-@args String span, String name, String labelKey, String helpKey, boolean optional
+@args String span, String name, String labelKey, String helpKey, boolean optional, boolean adminonly
```

Comment:
additionalClass as String using "admin-link" as parameter would be more re-usable